__tests__/domain/RacingGame/RacingGame.test.js

```diff
@@ -0,0 +1,72 @@
+import { RacingGame } from "../../../src/domain/RacingGame/RacingGame";
+import {
+  MIN_GAME_SIZE,
+} from "../../../src/domain/RacingGame/RacingGame.const";
+import {
+  START_POSITION,
+} from "../../../src/domain/Car/Car.const";
+import {
+  CAN_GO_ROUNDS,
+  DEFAULT_CAR_NAMES_INPUT,
+  DEFAULT_GAME_SIZE,
+  DEFAULT_PLAYERS, EXPECT_POSITION_LOGS,
+  EXPECT_POSITIONS, EXPECT_WINNER_LOG, EXPECT_WINNERS, RANDOM_NUMBERS_ALL
+} from "./RacingGame.test.const";
+
+describe("Together RacingGame (여럿이서 하는 레이싱 게임)", () => {
+  let racingGame = new RacingGame();
+  racingGame.setRacingGameSize(DEFAULT_GAME_SIZE);
+
+  it(`자동차 게임 시도 횟수를 입력받아서 자동차 게임을 셋팅할 때, 이상한 값을 입력했을 경우 에러를 출력하고 게임을 종료한다.`, () => {
+    expect(() => racingGame.setRacingGameSize('abc')).toThrow(`레이싱 게임 시도 회수를 숫자로 제대로 입력해야합니다.`);
+  })
+
+  it(`자동차 게임 시도 횟수를 입력받아서 자동차 게임을 셋팅할 때, 입력한 시도 횟수가 0 미만이면 에러를 출력하고 게임을 종료한다.`, () => {
+    expect(() => racingGame.setRacingGameSize(-1)).toThrow(`레이싱 게임 시도 회수는 ${MIN_GAME_SIZE} 이상이어야 합니다.`);
+  })
+
+  it(`"${DEFAULT_CAR_NAMES_INPUT}"가 레이싱 게임에 등록했을 때, 등록된 자동차는 총 ${DEFAULT_PLAYERS.length}대 이다.`, () => {
+    racingGame.setPlayers(
+      racingGame.getPlayerNamesFromInput(DEFAULT_CAR_NAMES_INPUT)
+    );
+    const expectedPlayersNames = DEFAULT_PLAYERS.map((player) =>
+      player.getName()
+    );
+    const receivedPlayersNames = racingGame
+      .getPlayers()
+      .map((player) => player.getName());
+    expect(receivedPlayersNames).toStrictEqual(expectedPlayersNames);
+  });
```

Comment:
`describe`을 한번 더 활용해서 context에 대한 부분을 따로 명시해도 좋을거 같네요!

```suggestion
describe("Together RacingGame (여럿이서 하는 레이싱 게임)", () => {
  let racingGame = new RacingGame();
  racingGame.setRacingGameSize(DEFAULT_GAME_SIZE);
  
  describe("자동차 게임 시도 횟수를 입력받았을 때", () => {
    it(`값이 올바르지 않은경우 에러를 출력하고 게임을 종료한다.`, () => {})
    it(`0 미만의 값이 입력 되면 에러를 출력하고 게임을 종료한다.', () => {})
  }
```